my_hairdressing_app/appointment_schedule_manager.py

```diff
@@ -0,0 +1,57 @@
+from my_hairdressing_app.logging import Logging
+from my_hairdressing_app.appointment_schedule import AppointmentSchedule
+from my_hairdressing_app.errors import BadDateFormatError, ShiftNotExistError
+
+class AppointmentScheduleManager:
+    """
+    Entidad manejadora del logging en el proceso de la aplicación.
+
+    Methods
+    -------
+    book_appointment(self, date: str, shift, appointment)
+    """
+    def __init__(self, test = False):
+        """
+        Constructor de la entidad.
+
+        Parameters
+        ----------
+        None
+        """
+        self.schedule = AppointmentSchedule()
+        self.logger = Logging(__name__, test).get_logger()
```

Comment:
No estás usando la configuración, ni estás usando un objeto externo. Por *separation of concerns*, además, esta clase está encargada de crear el logger, lo que no es posible.

Reply:
Vale creo que ya he entendido, yo lo que debo hacer es importar el logging ya configurado y usarlo directamente.